src/main/scala/cromwell/engine/backend/sge/SgeBackend.scala

```diff
@@ -181,7 +181,7 @@ class SgeBackend extends Backend with SharedFileSystem {
         FailedExecution(new Exception(message), Option(0))
       case (r, _) =>
         postProcess(backendCall) match {
-          case Success(callOutputs) => SuccessfulExecution(callOutputs, r)
+          case Success(callOutputs) => SuccessfulExecution(callOutputs, Seq.empty, r)
```

Comment:
My point was should you have Seq.empty[ExecutionEventEntry] so that it's not Seq[Nothing]. If downstream really doesn't care that's fine.